dali/operators/reader/loader/loader.h

```diff
@@ -352,6 +353,9 @@ class Loader {
   // If true, the last batch will be padded with the last sample so that the number
   // of samples matches batch size
   bool pad_last_batch_;
+  // If true data will be always read using read function and copied instead of shared with the
+  // target tensor
+  bool dont_map_files_;
```

Comment:
It's a matter of personal preference, I guess, but I think that it's better to use non-negated flags - it's a class with non-trivial constructor, so we can initialize this flag to whatever value we want.

Reply:
Done